libyul/AsmJsonImporter.cpp

```diff
@@ -197,13 +200,16 @@ Literal AsmJsonImporter::createLiteral(Json const& _node)
 	{
 		lit.kind = LiteralKind::String;
 		yulAssert(
-			lit.value.str().size() <= 32,
-			"String literal too long (" + std::to_string(lit.value.str().size()) + " > 32)"
+			value.size() <= 32,
+			"String literal too long (" + std::to_string(value.size()) + " > 32)"
 		);
+		unlimited = member(_node, "unlimited").get<bool>();
```

Comment:
You'll need a changelog entry for this, since it's a visible change (i.e. user facing).

Reply:
fair point, i postponed doing anything changelog related before it's clear that everyone agrees this is the right approach